-v/--verbosity switch doesn't work on Unix

```
vihofer@DESKTOP-CC8UHUD:~/git/aspnetcore$ ./eng/build.sh -v detailed
warning: No default group of projects was specified, so building the 'managed' and its dependent subset of projects. Run build.sh --help for more details.
warning: Building of NodeJS projects is disabled since node is not detected on Path and no BuildNodeJs or NoBuildNodeJs setting is set explicitly.
warning: Some managed projects depend on NodeJS projects. Building NodeJS is disabled so the managed projects will fallback to using the output from previous builds. The output may not be correct or up to date.
MSBUILD : error MSB1016: Specify the verbosity level.
    Full command line: '/home/vihofer/git/aspnetcore/.dotnet/sdk/10.0.100-preview.4.25211.22/MSBuild.dll -maxcpucount -verbosity:m -tlp:default=auto /m /nologo /clp:Summary /v:minimal /nr:true /warnaserror /p:TreatWarningsAsErrors=true /p:ContinuousIntegrationBuild=false /home/vihofer/.nuget/packages/microsoft.dotnet.arcade.sdk/10.0.0-beta.25214.8/tools/Build.proj -p:RepoRoot=/home/vihofer/git/aspnetcore/ -p:Projects=/home/vihofer/git/aspnetcore/eng/tools/RepoTasks/RepoTasks.csproj -p:Configuration=Release -p:Restore=true -p:Build=true -clp:NoSummary -v detailed -distributedlogger:Microsoft.DotNet.Cli.Commands.MSBuild.MSBuildLogger,/home/vihofer/git/aspnetcore/.dotnet/sdk/10.0.100-preview.4.25211.22/dotnet.dll*Microsoft.DotNet.Cli.Commands.MSBuild.MSBuildForwardingLogger,/home/vihofer/git/aspnetcore/.dotnet/sdk/10.0.100-preview.4.25211.22/dotnet.dll'
  Switches appended by response files:
Switch: -v

For switch syntax, type "MSBuild -help"
Build failed with exit code 1. Check errors above.
vihofer@DESKTOP-CC8UHUD:~/git/aspnetcore$ ./eng/build.sh --verbosity detailed
warning: No default group of projects was specified, so building the 'managed' and its dependent subset of projects. Run build.sh --help for more details.
warning: Building of NodeJS projects is disabled since node is not detected on Path and no BuildNodeJs or NoBuildNodeJs setting is set explicitly.
warning: Some managed projects depend on NodeJS projects. Building NodeJS is disabled so the managed projects will fallback to using the output from previous builds. The output may not be correct or up to date.
MSBUILD : error MSB1016: Specify the verbosity level.
    Full command line: '/home/vihofer/git/aspnetcore/.dotnet/sdk/10.0.100-preview.4.25211.22/MSBuild.dll -maxcpucount -verbosity:m -tlp:default=auto /m /nologo /clp:Summary /v:minimal /nr:true /warnaserror /p:TreatWarningsAsErrors=true /p:ContinuousIntegrationBuild=false /home/vihofer/.nuget/packages/microsoft.dotnet.arcade.sdk/10.0.0-beta.25214.8/tools/Build.proj -p:RepoRoot=/home/vihofer/git/aspnetcore/ -p:Projects=/home/vihofer/git/aspnetcore/eng/tools/RepoTasks/RepoTasks.csproj -p:Configuration=Release -p:Restore=true -p:Build=true -clp:NoSummary --verbosity detailed -distributedlogger:Microsoft.DotNet.Cli.Commands.MSBuild.MSBuildLogger,/home/vihofer/git/aspnetcore/.dotnet/sdk/10.0.100-preview.4.25211.22/dotnet.dll*Microsoft.DotNet.Cli.Commands.MSBuild.MSBuildForwardingLogger,/home/vihofer/git/aspnetcore/.dotnet/sdk/10.0.100-preview.4.25211.22/dotnet.dll'
  Switches appended by response files:
Switch: --verbosity

For switch syntax, type "MSBuild -help"
Build failed with exit code 1. Check errors above
```

--- a/eng/build.sh
+++ b/eng/build.sh
@@ -232,6 +232,11 @@ while [[ $# -gt 0 ]]; do
         -excludeCIBinarylog|-nobl)
             exclude_ci_binary_log=true
             ;;
+        -verbosity|-v)
+            shift
+            [ -z "${1:-}" ] && __error "Missing value for parameter --verbosity" && __usage
+            verbosity="${1:-}"
+            ;;
         -dotnet-runtime-source-feed|-dotnetruntimesourcefeed|-runtime-source-feed|-runtimesourcefeed)
             shift
             [ -z "${1:-}" ] && __error "Missing value for parameter --runtime-source-feed" && __usage

